cluster/examples/kubernetes/ceph/object-test.yaml

```diff
@@ -22,3 +22,7 @@ spec:
     port: 80
     securePort:
     instances: 1
+  multisite:
```

Comment:
Instead of modifying the object-test.yaml example, let's create a new object-multisite.yaml. Let's keep object.yaml and object-test.yaml as the basic case where there is no multisite.